backend/controllers/appointments/addAppointment.ts

```diff
@@ -131,22 +143,17 @@ export const addAppointmentFactory = (appointmentsService, existingEventsService
   const renderTemplate = async (req, res, pageData) => {
     const { formValues } = pageData || {}
     const { offenderNo } = req.params
-    const { activeCaseLoadId } = req.session.userDetails
 
     try {
-      const prePopulatedData = {}
-      const { firstName, lastName, bookingId } = await prisonApi.getDetails(res.locals, offenderNo)
+      const prePopulatedData: PrePopulatedData = {}
+      const { firstName, lastName, bookingId, agencyId } = await prisonApi.getDetails(res.locals, offenderNo)
       const offenderName = `${properCaseName(lastName)}, ${properCaseName(firstName)}`
-      const { appointmentTypes, appointmentLocations } = await getAppointmentTypesAndLocations(
-        res.locals,
-        activeCaseLoadId
-      )
+      const { appointmentTypes, appointmentLocations } = await getAppointmentTypesAndLocations(res.locals, agencyId)
```

Comment:
So we were passing the wrong thing in before - should have been agencyId?

Reply:
yep - apparently it wasn't an issue before as users would only be booking prisoners into appointments and they'd both be in the same caseload. But now with video booking they often have to book appointments for prisoner's in other caseloads